app/src/playStore/java/org/fossasia/openevent/app/core/attendee/ScanningDecider.java

```diff
@@ -14,4 +22,16 @@ public void openScanQRActivity(Context context, long eventId) {
         intent.putExtra(MainActivity.EVENT_KEY, eventId);
         context.startActivity(intent);
     }
+
+    public static List<Object[]> getActivitiesDataArray() {
```

Comment:
No need of duplicating, just remove the ScanQRActivity from the list, we are not using instantiation tests anyway

Reply:
Okay